third-party/linshare/src/test/java/org/apache/james/linshare/LinshareConfigurationTest.java

```diff
@@ -40,51 +43,57 @@ void shouldMatchBeanContract() {
     @Test
     void fromShouldThrowWhenUrlIsNull() {
         PropertiesConfiguration configuration = new PropertiesConfiguration();
-        configuration.addProperty("blob.export.linshare.token", "token");
-        configuration.addProperty("blob.export.linshare.url", null);
+        configuration.addProperty(LinshareConfiguration.USER_NAME_PROPERTY, LinshareFixture.TECHNICAL_ACCOUNT.getUsername());
+        configuration.addProperty(LinshareConfiguration.PASSWORD_PROPERTY, LinshareFixture.TECHNICAL_ACCOUNT.getPassword());
+        configuration.addProperty(LinshareConfiguration.URL_PROPERTY, null);
 
         assertThatThrownBy(() -> LinshareConfiguration.from(configuration)).isInstanceOf(MalformedURLException.class);
     }
 
     @Test
-    void fromShouldThrowWhenTokenIsNull() {
+    void fromShouldThrowWhenBasicAuthnIsNull() {
         PropertiesConfiguration configuration = new PropertiesConfiguration();
-        configuration.addProperty("blob.export.linshare.token", null);
-        configuration.addProperty("blob.export.linshare.url", "http://127.0.0.1:8080");
+        configuration.addProperty(LinshareConfiguration.USER_NAME_PROPERTY, null);
+        configuration.addProperty(LinshareConfiguration.PASSWORD_PROPERTY, null);
+        configuration.addProperty(LinshareConfiguration.URL_PROPERTY, DEFAULT_URL);
 
-        assertThatThrownBy(() -> LinshareConfiguration.from(configuration)).isInstanceOf(IllegalArgumentException.class);
+        assertThatThrownBy(() -> LinshareConfiguration.from(configuration)).isInstanceOf(NullPointerException.class);
     }
 
     @Test
     void fromShouldThrowWhenURLIsInvalid() {
         PropertiesConfiguration configuration = new PropertiesConfiguration();
-        configuration.addProperty("blob.export.linshare.token", "token");
-        configuration.addProperty("blob.export.linshare.url", "invalid");
+        configuration.addProperty(LinshareConfiguration.USER_NAME_PROPERTY, LinshareFixture.TECHNICAL_ACCOUNT.getUsername());
+        configuration.addProperty(LinshareConfiguration.PASSWORD_PROPERTY, LinshareFixture.TECHNICAL_ACCOUNT.getPassword());
+        configuration.addProperty(LinshareConfiguration.URL_PROPERTY, "invalid");
 
         assertThatThrownBy(() -> LinshareConfiguration.from(configuration)).isInstanceOf(MalformedURLException.class);
     }
 
     @Test
-    void fromShouldThrowWhenTokenIsEmpty() {
+    void fromShouldThrowWhenBasicAuthnIsEmpty() {
```

Comment:
`Authn` ?

Reply:
Right, it should be **Auth** for Authentication